cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -270,11 +270,15 @@ def _repr_svg_(self) -> str:
         # coverage: ignore
         _validate_circuit(self.circuit)
         tdd = self.circuit.to_text_diagram_drawer(transpose=False)
+        if len(tdd.horizontal_lines) == 0:
+            return '<svg></svg>'
         return tdd_to_svg(tdd)
 
 
 def circuit_to_svg(circuit: 'cirq.Circuit') -> str:
     """Render a circuit as SVG."""
     _validate_circuit(circuit)
     tdd = circuit.to_text_diagram_drawer(transpose=False)
+    if len(tdd.horizontal_lines) == 0:
```

Comment:
why do we need this again?

Reply:
In circuits with no non-empty moments, this will return a blank SVG. (this code snippet was redundant as the same code was present in _repr_svg_ ). I have added a comment describing the case.